governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+# Maker Executive Aug 2 2023
+
+# [Executive Proposal] DSR Increase, Spark D3M Changes, Harbor Trade Credit (RWA004-A) Changes, New Silver (RWA002-A) Changes, Launch Project Funding, AVC Member Participation Rewards, Clydesdale Quarterly Surplus Return - August 2, 2023
+
+The Governance Facilitator(s), AmusingAxl, Sidestream and Arby have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

Comment:
AmusingAxl is part of Dewiz, who we'd usually list as a group rather than as an individual. Arby we seemingly don't talk about? Not entirely sure why. Phoenix Labs also contributed the Spark changes, I believe.